examples/alarm/Kconfig

```diff
@@ -33,6 +33,6 @@ config EXAMPLES_ALARM_DEVPATH
 
 config EXAMPLES_ALARM_SIGNO
 	int "Alarm signal"
-	default 1
```

Comment:
Maybe we can default to `SIGRTMIN`?
or maybe add some range check to make sure that this is a realtime signal

Reply:
Can this be resolved?  This PR does not address the Kconfig signal number settings any more.

Reply:
Done.